client/src/luigi-client.ts

```diff
@@ -0,0 +1,180 @@
+import {
+  AuthData,
+  ClientPermissions,
+  Context,
+  CoreSearchParams,
+  LinkManager,
+  NodeParams,
+  PathParams,
+  StorageManager,
+  UserSettings,
+  UxManager
+} from '../luigi-client';
```

Comment:
whats the use of these imports here ? They dont seem to be used in the file. I am guessing it also messes up with docu generation somehow

Reply:
These are interfaces used for method and argument typings